omod/src/main/webapp/admin/concepts/conceptDrugForm.jsp

```diff
@@ -10,9 +10,8 @@
 <script type="text/javascript">
 	dojo.require("dojo.widget.openmrs.ConceptSearch");
 	dojo.require("dojo.widget.openmrs.OpenmrsPopup");
-
 	<request:existsParameter name="autoJump">
-		var autoJump = <request:parameter name="autoJump"/>;
+		var autoJump = ${openmrs:getSafeJsBoolean(param.autoJump)};
```

Comment:
Yeah, no problem

Take this request:
`conceptDrug.form?autoJump=test'alert(1)`

If the JSP were using <c:out> tags, `test'alert(1)` would turn into `test&#39;alert(1)` and would generate the following JS:
`var autoJump = test&#39;alert(1);`